media/views/oss/suggestion/suggestion.css

```diff
@@ -0,0 +1,141 @@
+html {
+  height: 100%;
+  font-size: 62.5%;
+}
+body {
+  height: 100%;
+  padding: 0;
+  font-size: 1.4rem;
+  line-height: 1.5;
+}
+body * {
+  box-sizing: border-box;
+}
+
+h2 {
+  font-size: 1em;
+  margin-block-start: 0;
+}
+
+a {
+  cursor: pointer;
+  color: #7c74f2;
+  user-select: none;
```

Comment:
any reason we're not allowing users to select the text? in case they want to copy it

Reply:
No reason, the code was refactored from div to accessible anchor element but css not adjusted. Great spot @sharonsit ! :)